medusa/failed_processor.py

```diff
@@ -37,11 +29,11 @@ def __init__(self, dirName, nzbName):
 
     def process(self):
         """
-        Do the actual work
+        Do the actual work.
 
         :return: True
         """
-        self._log(u'Failed download detected: ({nzb}, {dir})'.format(nzb=self.nzb_name, dir=self.dir_name))
+        self._log(u'Failed download detected: ({nzb}, {dir})', {'nzb': self.nzb_name, 'dir': self.dir_name})
```

Comment:
@fernandog I'm not proposing changing the functionality of `self._log` just the name.  It appends `\n` to every log line regardless of log level.  I believe that @p0psicles is proposing a log adapter so we can do `self.log.debug(msg)` type logging.  While I would normally agree, the log adapter would have such limited use I think the single 2-line method that currently exists is sufficient and adding an adapter here isn't necessary.